src/KeyringController.ts

```diff
@@ -721,40 +908,28 @@ class KeyringController extends EventEmitter {
    * matches the provided `type`,
    * returning it if it exists.
    *
-   * @param {string} type - The type whose class to get.
-   * @returns {Keyring|undefined} The class, if it exists.
+   * @param type - The type whose class to get.
+   * @returns The class, if it exists.
    */
-  getKeyringBuilderForType(type) {
+  #getKeyringBuilderForType(type: string): any {
```

Comment:
Instead of `any` should this be `{ (): Keyring<Json>; type: string }`?

Reply:
Improved in [ce7d126](https://github.com/MetaMask/KeyringController/pull/202/commits/ce7d126375790bd07789a82f6c9609f0a2eb9813).